paasta_tools/cli/cmds/local_run.py

```diff
@@ -462,6 +463,30 @@ def add_subparser(subparsers):
         action="store_true",
         default=False,
     )
+    list_parser.add_argument(
+        "--assume-role",
+        help="role ARN to assume before launching the service",
```

Comment:
Would help to show an example ARN (eg full `arn:aws:iam::...:role/...`) for what's expected, based on the param name `assume-role` my first instinct would be to just erroneously try to give the role name without something reminding me of the full ARN format.

Reply:
I feel that using `aws-okta` and letting the end-user choose the account / role / etc. interactively might be an easier interface for most than forcing folks to specify the full role-arn. Having said that, I haven't actually reviewed any of the code yet :)